graylog2-web-interface/src/components/indices/IndexSetTemplates/TemplateDetails.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { PluginStore } from 'graylog-web-plugin/plugin';
+
+import { Col, Row } from 'components/bootstrap';
+import { Section, Icon } from 'components/common';
+import IndexMaintenanceStrategiesSummary from 'components/indices/IndexMaintenanceStrategiesSummary';
+import { DataTieringSummary, DATA_TIERING_TYPE } from 'components/indices/data-tiering';
+import type { IndexSetTemplate } from 'components/indices/IndexSetTemplates/types';
+
+type Props = {
+  template: IndexSetTemplate,
+}
+
+const FlexWrapper = styled.div(({ theme }) => css`
+  display: flex;
+  flex-direction: column;
+  gap: ${theme.spacings.md};
+
+    dl {
+      margin-bottom: 0;
+    }
+
+    dl > dd {
+      margin-bottom: ${theme.spacings.sm};
+    }
+
+    dl > dd:last-child {
+      margin-bottom: 0;
+    }
+`);
+
+const RotationSummaryWrapper = styled.div(({ theme }) => css`
+  margin-bottom: ${theme.spacings.sm};
+`);
+
+const WarmTierSummaryWrapper = styled.div(({ theme }) => css`
+  margin-top: ${theme.spacings.sm};
+`);
+
+const formatRefreshInterval = (intervalInMs : number) => {
+  const intervalInSeconds = intervalInMs / 1000;
+
+  if (intervalInSeconds > 60) {
+    return `${intervalInSeconds / 60} minutes`;
+  }
+
+  return `${intervalInSeconds} seconds`;
+};
+
+const TemplateDetails = ({
+  template,
+}: Props) => {
+  const dataTieringPlugin = PluginStore.exports('dataTiering').find((plugin) => (plugin.type === DATA_TIERING_TYPE.HOT_WARM));
+
+  return (
+    <Row>
+      <Col md={12}>
+        <FlexWrapper>
+          <Section title="Details">
+            <dl>
+              <dt>Index Analyzer:</dt>
```

Comment:
In general I agree. However, for this view I am reusing the existing summary components for Data Tiering and all the legacy Retention and Rotation. Those are also used on the index detail. They have the same layout (value below label). I think it is more important to be consistent that's why I aligned the other attributes the same way. I also don't want to copy-paste the summary components just for this but rather reuse them. We can definitively think about changing the layout for those overall, as it is not pretty. But I think this is out of scope for this PR.

Reply:
Thank you for having a look, I think it would be great to have it aligned like below
![image](https://github.com/Graylog2/graylog2-server/assets/1381455/c8998efc-a0b5-44f4-bd4f-5806ea3f290d)

I agree we should make this content consistent by updating the logic of the Summary component.